docs/articles/documentation/how-it-works/README.md

```diff
@@ -33,12 +31,12 @@ Every TestCafe test begins with an empty slate of a browser. At the end of the r
 
 Tests that run in parallel operate in independent sandboxed environments. This helps prevent server-side collisions.
 
-## Arbitrary Client-Side Injections
+## Client-Side Scripts
 
-To perform common testing tasks, TestCafe translates server-side calls to its API into client-side code. However, some scenarios require the execution of client-side code that TestCafe cannot automatically generate. That’s why the framework provides several ways to execute arbitrary JavaScript.
+To perform common testing tasks, TestCafe translates server-side calls to its API into client-side code. However, some scenarios require the execution of client-side code that TestCafe cannot automatically generate. That’s why the framework provides several ways to execute user-defined JavaScript.
 
 [Client Scripts](/../guides/advanced-guides/inject-client-scripts.md) inject custom JavaScript files into the page.
-[Client Functions](../guides/basic-guides/obtain-client-side-info.md) evaluate arbitrary JavaScript expressions and pass their return value to the server side.
+[Client Functions](../guides/basic-guides/obtain-client-side-info.md) evaluate user-defined JavaScript expressions and pass their return value to the server side.
```

Comment:
This information does not make it clear on why and when use Client Functions.

Reply:
Neither does the previous line: this list, as I imagined it, was not meant to describe use cases, but rather make the reader aware of the opportunities they have when it comes to client-side injections. Should I change both lines to fit this approach?

Reply:
I would change them. Let's ask others for their opinion.